File: things/src/main/java/com/krisbiketeam/smarthomeraspbpi3/utils/LogConsole.java
package com.krisbiketeam.smarthomeraspbpi3.utils;

import android.databinding.BaseObservable;
import android.databinding.Bindable;

import com.krisbiketeam.smarthomeraspbpi3.BR;

public class LogConsole extends BaseObservable {

    private String consoleMessage;
    
    @Bindable
    public String getConsoleMessage() {
        return consoleMessage;
    }

    public void setConsoleMessage(String consoleMessage) {
        this.consoleMessage = consoleMessage;
        notifyPropertyChanged(BR.consoleMessage);
    }
}

File: things/src/main/java/com/krisbiketeam/smarthomeraspbpi3/utils/LogConsole.kt
package com.krisbiketeam.smarthomeraspbpi3.utils

import androidx.databinding.BaseObservable
import androidx.databinding.Bindable

import com.krisbiketeam.smarthomeraspbpi3.BR

class LogConsole : BaseObservable() {

    @get:Bindable
    var consoleMessage: String? = null
        set(consoleMessage) {
            field = consoleMessage
            notifyPropertyChanged(BR.consoleMessage)
        }
}
